gui.js

```diff
@@ -0,0 +1,93 @@
+// defined in /js/player.js
```

Comment:
It should be defined in js/gui.js and not here.

Reply:
sorry, in my local project the character file is in the character subfolder and the gui.js is in the js-folder. It is a problem in git handling because I never before used git.

Reply:
You can rename it in your pull request. Let us know if you need help. No problem!